scripts/format.py

```diff
@@ -22,7 +22,8 @@
                  'release.h', 'genrand.cpp', 'address.cpp', 'visualizer_constants.hpp', 'icu-collate.cpp', 'icu-collate.hpp',
                  'yyjson.cpp', 'yyjson.hpp', 'duckdb_pdqsort.hpp', 'stubdata.cpp',
                  'nf_calendar.cpp', 'nf_calendar.h', 'nf_localedata.cpp', 'nf_localedata.h', 'nf_zformat.cpp',
-                 'nf_zformat.h', 'expr.cc', 'function_list.cpp']
+                 'nf_zformat.h', 'expr.cc', 'function_list.cpp', 'serialize_parsed_expression.cpp',
+                 'serialize_result_modifier.cpp', 'serialize_query_node.cpp']
```

Comment:
Could it make sense to have as part of the auto-generated header something saying "SKIP ME FROM FORMAT" or equivalent, and use that for pattern matching (on top of this list)?